src/util/amount.rs

```diff
@@ -1247,7 +1247,6 @@ mod tests {
     #[cfg(feature = "serde")]
     #[test]
     fn serde_as_sat() {
-        use serde::{Deserialize, Serialize};
 
         #[derive(Serialize, Deserialize, PartialEq, Debug)]
```

Comment:
How does that derivation still work then??

Reply:
Because we're not using edition 2018.

in 2015 edition what's important is `#[macro_use]` Which we still have here:
https://github.com/rust-bitcoin/rust-bitcoin/blob/master/src/lib.rs#L55

Reply:
Ah true.